### ISSUE TITLE
Unable to see break even point in Rate optmization workbook 0.4

### ISSUE THREAD

**Issue by pankaj392** [microsoft/finops-toolkit#838]
Hello Team,

I just upgraded by optimization workbook to 0.4 version. As mentioned in the changelogs there is newly Added reservation recommendations with the break-even point to identify when savings would be achieved.

Unfortunately I see same page is it was with 0.3 version. Can you please guide me where should I see this feature.

Steps to Reproduce.
1. Open Cost Optimization workbook 0.4
2. Navigate to Tab rate optimization
3. Navigate to Azure reservations
4.I only see reservation summary for 1year Payment Term and 3 year Payment term along and other details like subscription details etc.

No where I see break even point summary. Am I missing something or am I in wrong screen.

Can you please guide me.

Thanks and regards,
Pankaj

**Comment by arthurclares** [comment]
@pankaj392 Can you please compare the version that you deployed with the version of the Cost Optimization workbook that is available on Azure Advisor? They should match. 
The Toolkit version is an editable version of the Advisor workbook.

**Comment by pankaj392** [comment]
@arthurclares : While navigating to the Advisor workbook, I noticed a new version named Cost Optimization (Preview). Additionally, when navigating to the Rate Optimization tab, I observed a new tab labeled Azure Reservation (Preview). In this new preview tab, there is a Break Even Point column for each SKU.

While deploying cost optimitaion workbook through finops tool kit, this preview tab is missing.

<img width="786" alt="AdvisorWorkbook" src="https://github.com/user-attachments/assets/f1535f2f-1f4c-4905-9f02-63340bc68350">
